src/modules/Login/action.js

```diff
@@ -181,3 +193,21 @@ export const logout = (username?: string) => ({
   type: Constants.LOGOUT,
   data: { username }
 })
+
+const getReceiveAddresses = (currencyWallets: { [id: string]: EdgeCurrencyWallet }): Promise<{ [id: string]: EdgeReceiveAddress }> => {
```

Comment:
This is the same code as in `src/modules/Core/Wallets/action.js` so you can just export/import the function from there.
But I don't understand why even get addresses on login? Why not just get them when we request for one instead of loading them in advance?